src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/experimental_detectron_topk_rois_ref.cl

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "include/batch_headers/common.cl"
+#include "include/batch_headers/data_types.cl"
+
+KERNEL(experimental_detectron_topk_rois_ref)(const __global INPUT0_TYPE* input_rois,
+        const __global INPUT1_TYPE* topk_indices, __global OUTPUT_TYPE* output_rois)
+{
+    const uint b = get_global_id(0);
+    const uint output_idx = OUTPUT_GET_INDEX(b, 0, 0, 0);
+    const uint roi_idx = topk_indices[b];
+    const input_idx = INPUT0_GET_INDEX(roi_idx, 0, 0, 0);
+    output_rois[output_idx] = input_rois[input_idx];
+    output_rois[output_idx + 1] = input_rois[input_idx + 1];
+    output_rois[output_idx + 2] = input_rois[input_idx + 2];
+    output_rois[output_idx + 3] = input_rois[input_idx + 3];
```

Comment:
Since it supports differnt input/output types, it needs to convert types b/w input & output

Reply:
it's expected from operation definition that input and output types are the same

Reply:
I can use INPUT0_TYPE define for output type also to point this more direct, but using OUTPUT_TYPE is common approach for other kernels also where types are the same, isn't it?